src/sdk/module/default/simple_login_manager.cc

```diff
@@ -0,0 +1,20 @@
+#include "dsa_common.h"
+
+#include "simple_login_manager.h"
+
+#include "../../util/client_info.h"
+#include "../../web_server/login_manager.h"
+
+namespace dsa {
+
+void SimpleLoginManager::check_login(const string_ &username,
+                                     const string_ &password,
+                                     ClientInfo::GetClientCallback &&callback) {
+}
+
+void SimpleLoginManager::get_user(const string_ &username,
+                                  ClientInfo::GetClientCallback &&callback) {
+  // TODO check username and set dsid for client info accordingly
+  callback(ClientInfo(), false);
```

Comment:
username should be set as the id of the ClientInfo output
also it's better to make the callback async